client/src/test/scala/org/http4s/client/ClientRouteTestBattery.scala

```diff
@@ -8,35 +8,31 @@ import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
 import org.http4s.Uri.{Authority, RegName}
 import org.http4s.client.testroutes.GetRoutes
 
+import org.specs2.mutable.Specification
 import org.specs2.specification.core.{ Fragments, Fragment }
 
 import scalaz.concurrent.Task
 import scalaz.stream.Process
 
 
 abstract class ClientRouteTestBattery(name: String, client: Client)
-  extends JettyScaffold(name) with GetRoutes
+  extends Http4sSpec with JettyScaffold with GetRoutes
 {
-  // Travis has been timing out intermittently.  Let's see if having All The Threads helps.
-  sequential
-  isolated
```

Comment:
Nice to see those two go.  Comment makes them sound like an attempt to work around a deeper problem.